src/mabrobad.c

```diff
@@ -0,0 +1,848 @@
+		{10, -1},
+		{-1},
+		{-1},
+		{-1}
+	},
+	{ // 7
+		{4, 2, -1},
+		{8, 9, -1},
+		{11, -1},
+		{10, -1},
+		{6, -1},
+		{3, -1}
+	},
+	{ // 8
+		{5, -1},
+		{9, -1},
+		{12, -1},
+		{11, -1},
+		{7, 6, -1},
+		{4, 1, -1}
+	},
+	{ // 9
+		{12, -1},
+		{8, 7, 6, -1},
+		{5, 2, -1},
+		{-1},
+		{-1},
+		{-1}
+	},
+	{ //10
+		{7, 4, 2, -1},
+		{11, 12, -1},
+		{6, -1}, 
+		{-1},
+		{-1},
+		{-1}
+	},
+	{ //11
+		{8, 5, -1},
+		{12, -1},
+		{10, -1},
+		{7, 3, -1},
+		{-1},
+		{-1} 
+	},
+	{ //12
+		{9, -1},
+		{11, 10, -1},
+		{8, 4, 1, -1},
+		{-1},
+		{-1},
+		{-1}
+	}
+};
+
+
+void decode_move(MOVE move, int* start, int* end);
+POSITION DoMove(POSITION position, MOVE move);
+void genHelper(char* board, int i, MOVELIST **moves);
+BOOLEAN checkAdj(char* board, int i, int player);
+void DFS(int node, char piece, BOOLEAN visited[], char board[BOARDSIZE]);
+
+CONST_STRING kAuthorName = "Aditya Tummala";
+CONST_STRING kGameName = "Abrobad";
+CONST_STRING kDBName = "abrobad";
+
+char pieces[] = "-XO";
+
+/**
+ * @brief An upper bound on the number of reachable positions.
+ *
+ * @details The hash value of every reachable position must be less
+ * than `gNumberOfPositions`.
+ */
+
+POSITION gNumberOfPositions = 0;
+
+/**
+ * @brief The hash value of the initial position of the default
+ * variant of the game.
+ * 
+ * @note If multiple variants are supported and the hash value
+ * of the initial position is different among those variants,
+ * then ensure that gInitialPosition is modified appropriately
+ * in both setOption() and GameSpecificMenu(). You may also
+ * choose to modify `gInitialPosition` in InitializeGame().
+ */
+POSITION gInitialPosition = 0;
+
+/**
+ * @brief Indicates whether this game is PARTIZAN, i.e. whether, given
+ * a board, each player has a different set of moves 
+ * available to them on their turn. If the game is impartial, this is FALSE.
+ */
+BOOLEAN kPartizan = FALSE;
+
+/**
+ * @brief Whether a tie or draw is possible in this game.
+ */
+BOOLEAN kTieIsPossible = TRUE;
+
+/**
+ * @brief Whether the game is loopy. It is TRUE if there exists a position
+ * P in the game such that, there is a sequence of N >= 1 moves one can
+ * make starting from P that allows them to revisit P.
+ */
+BOOLEAN kLoopy = TRUE;
+
+/**
+ * @brief Whether symmetries are supported, i.e., whether there is
+ * at least one position P in the game that gCanonicalPosition() 
+ * maps to a position Q such that P != Q. If gCanonicalPosition
+ * (initialized in InitializeGame() in this file), is set to NULL,
+ * then this should be set to FALSE.
+ */
+BOOLEAN kSupportsSymmetries = FALSE;
+
+/**
+ * @brief Useful for some solvers. Do not change this.
+ */
+POSITION kBadPosition = -1;
+
+/**
+ * @brief For debugging, some solvers print debug statements while
+ * solving a game if kDebugDetermineValue is set to TRUE. If this
+ * is set to FALSE, then those solvers do not print the debugging
+ * statements.
+ */
+BOOLEAN kDebugDetermineValue = TRUE;
```

Comment:
Strongly recommend setting this to FALSE for production so that the solver will run faster in the future without printing all the debug messages.

Reply:
Great catch, thanks!